Login page fails to render: DatabaseMigrationService EJB throws 'Client not authorized' from unauthenticated footer

## Problem

The application fails to render the login page (`index1.xhtml`) on a fresh start. The server dispatches to `generalError.xhtml`, which cascades into the same failure, producing large stack traces in `server.log`.

### Root cause

`resources/ezcomp/footer.xhtml` (line 21) evaluates:

```xhtml
<h:panelGroup rendered="#{applicationController.databaseMigrationPending}">
```

This calls `ApplicationController.isDatabaseMigrationPending()` (line 156), which in turn calls the `@Singleton` EJB `DatabaseMigrationService.isMigrationPending()`.

Because the login page has no authenticated subject, Payara rejects the EJB call:

```
Caused by: javax.ejb.AccessLocalException: Client not authorized for this invocation
    at com.sun.ejb.containers.BaseContainer.preInvoke(BaseContainer.java:2013)
    ...
    at com.sun.proxy.$Proxy1186.isMigrationPending(Unknown Source)
    at com.divudi.bean.common.ApplicationController.isDatabaseMigrationPending(ApplicationController.java:156)
```

`DatabaseMigrationService` has no `@PermitAll` annotation. Other beans invoked from unauthenticated flows (e.g. `ConfigOptionFacade`, parts of `BillService` / `ChannelService`) are already annotated `@PermitAll`. This one is the odd one out.

### Impact

- Application is unusable — cannot reach the login page.
- The same footer is included in `generalError.xhtml`, so the container cannot even render its own error page.

## Proposed fix

Add `@PermitAll` to `DatabaseMigrationService` (class-level) so the public migration-status check is callable from unauthenticated rendering contexts. This matches the existing pattern used for `ConfigOptionFacade`.

```java
import javax.annotation.security.PermitAll;
...
@Singleton
@Startup
@PermitAll
public class DatabaseMigrationService { ... }
```

## Environment
- Payara 5.2022.5
- Branch: `development`

--- a/src/main/java/com/divudi/service/DatabaseMigrationService.java
+++ b/src/main/java/com/divudi/service/DatabaseMigrationService.java
@@ -14,6 +14,7 @@ import java.util.logging.Logger;
 import java.util.regex.Matcher;
 import java.util.regex.Pattern;
 import javax.annotation.PostConstruct;
+import javax.annotation.security.PermitAll;
 import javax.ejb.EJB;
 import javax.ejb.Singleton;
 import javax.ejb.Startup;
@@ -34,6 +35,7 @@ import javax.ejb.TransactionAttributeType;
  */
 @Singleton
 @Startup
+@PermitAll
 public class DatabaseMigrationService {
 
     private static final Logger LOGGER = Logger.getLogger(DatabaseMigrationService.class.getName());

